content/updates/2020-11/2-Import-and-export-your-reference-entity-records.md

```diff
@@ -0,0 +1,23 @@
+---
+pim_announcement_audience:
+- EE
+---
+
+# Import and export your reference entity records
+::: meta-data type="New" features="Productivity" available="late November"
+
+Having a hard time initializing the records of your reference entities? We got you covered! You can now import an Excel or CSV file containing your records. We also make it possible for you to export them in flat files, once enriched!
+
+Discover this new feature in your `Import` and `Export` menus, under the following job names: 
+- `Reference entity record import in XSLX`,
+- `Reference entity record import in CSV`,
+- `Reference entity record export in XLSX`,
+- `Reference entity record export in CSV`.
+
```

Comment:
```suggestion

This new feature can also help you bulk update your records or easily duplicate them.

```

Reply:
I'd suggest moving this sentence, after the picture.